src/makeInfinityAdder.js

```diff
@@ -4,7 +4,31 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (value) => {
+    if (value === undefined) {
+      return yieldAndReset();
+    } else {
+      return add(value);
+    }
+  };
+
+  const yieldAndReset = () => {
```

Comment:
Just for notice: 
it is unnecessary to create yieldAndReset and add functions. You didn't reuse them. It has made your code heavier.